src/AddTodoForm.jsx

```diff
@@ -1,13 +1,20 @@
+const AddTodoForm = (props) => {
 
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const todoTitle = event.target.title.value;
+    event.target.reset();
+    console.log(todoTitle);
+    props.onAddTodo(todoTitle);
+  }
 
-function AddTodoForm() {
   return (
-    <form>
+    <form onSubmit={handleAddTodo}>
       <label htmlFor="todoTitle">Title</label>
-      <input type="text" id="todoTitle" />
-      <input type="submit" value="Add"/>
+      <input type="text" id="todoTitle" name="title" />
+      <input type="submit" value="Add" />
```

Comment:
add a required attribute for validation

Reply:
Thank you! I added a required attribute for todoTitle